src/org/openlcb/hub/Hub.java

```diff
@@ -37,20 +37,40 @@ public class Hub {
     private final static Logger logger = Logger.getLogger(Hub.class.getName());
     public final static int DEFAULT_PORT = 12021;
     final static int CAPACITY = 20;  // not too long, to reduce delay
-    private final boolean lineEndings;
+    private final boolean sendLineEndings;
+    private final boolean receiveLineEndings;
```

Comment:
let's call this `requireReceivedLineEndings` or `requireIncomingLineEndings`

Reply:
Thanks, will update.